app/src/main/java/chat/rocket/android/userdetails/ui/UserDetailsFragment.kt

```diff
@@ -97,19 +97,23 @@ class UserDetailsFragment : Fragment(), UserDetailsView {
         requestBuilder.apply(RequestOptions.bitmapTransform(RoundedCorners(14)))
             .into(image_avatar)
 
-        text_name.text = name
-        text_username.text = username
-        text_description_status.text = status.substring(0, 1).toUpperCase() + status.substring(1)
-        text_description_timezone.text = utcOffset
+        text_name.text = name ?: ""
+        text_username.text = username ?: ""
 
-        // We should also setup the user details listeners.
-        text_message.setOnClickListener { presenter.createDirectMessage(username) }
+        val userStatus = if(status != null) status.substring(0, 1).toUpperCase() + status.substring(1) else  ""
+        text_description_status.text = userStatus
+
+        val userUtcOffset = if(utcOffset != "null") utcOffset else "None"
```

Comment:
Hard-coded string, please remove.

Reply:
@filipedelimabrito removed hard coded string and made a new string